apps/ejabberd/src/ejabberd_app.erl

```diff
@@ -55,7 +55,6 @@ start(normal, _Args) ->
     gen_mod:start(),
     ejabberd_config:start(),
     ejabberd_check:config(),
-    maybe_start_alarms(),
```

Comment:
1. They were not used anywhere after removing "watchdog" sending warnings over XMPP.
2. Wombat does more or else the same.